config/event-schema.json

```diff
@@ -49,6 +52,24 @@
     },
     "user_id": {
       "type": "integer"
+    },
+    "data": {
+      "type": "array",
+      "items": [
+        {
+          "type": "object",
+          "required": ["key", "value"]
+        }
+      ]
+    },
+    "event_tags": {
+      "anyOf": [
+        {
+          "type": "array",
+          "items" : [ { "type": "string" } ]
+        },
+        { "type": "null" }
```

Comment:
Isn't this essentially the same as saying the field is optional/not-required? Why do it two ways?

Reply:
Good point. I'll amend